Makefile

```diff
@@ -11,7 +11,7 @@ ifndef TEMP_DIR
 TEMP_DIR:=$(shell mktemp -d /tmp/heapster.XXXXXX)
 endif
 
-VERSION?=v2.0.0
+VERSION?=v2.0.1
```

Comment:
I believe this version is ignored in current release process.

Why was the version bumped from 1.7 to 2.0?

Reply:
I don't know. If we don't use it, I'll change this to "dev" so that pushing dev images works and that nobody will think of this as an actual version tag.